paper/paper.md

```diff
@@ -0,0 +1,481 @@
+Anyone can contribute to our educational materials [on GitHub](https://github.com/GWC-DCMB).
+
+
+# Statement of Need
+
+As women bioinformatics trainees at the
+University of Michigan (U-M), we experience the gender gap in our field first-hand.
+During the 1974-1975 academic year, women achieved 18.9% of total Bachelor's
+degrees in computer and information sciences in the US [@nces_digest_2012].
+By 1983-1984 this peaked at 37.1%, but fell to 17.6% by 2010-2011. We also see this
+national trend in the training of the next generation of Bioinformaticians at
+Michigan Medicine.
+Since accepting its first students in 2001, the U-M Bioinformatics Graduate
+Program has graduated 66 male and 22 female doctorates as of 2019.
+This disparity begins at the applicant level; during 2016-2019 the average
+percentage of females applying directly to the Bioinformatics PhD program was
+35.2%, and the average percentage of female applicants listing Bioinformatics as first or second
+choice in the Program in Biomedical Sciences, U-M's biomedical PhD umbrella
+program was 41%.
+
+Previous research on women's educational experiences in science, technology, engineering,
+and mathematics (STEM) have produced
+various explanations for persistent gender disparities
+[@benbow_gender_2016].
+One explanation is that women in first year computer science courses rate
+themselves as being less experienced, prepared, or able to master the course
+material than the men [@fisher_undergraduate_1997].
+Additionally, women often experience stereotype threats that negative
+influence their math and science performance and deter them from pursuing STEM as a career [@hill_why_2010].
+The majority of our organization's founding graduate students (all women) began
+coding in our undergraduate careers or later.
+We wanted to provide a safe place for local high school women to
+develop confidence in themselves and their computational skills
+before college, and be exposed to successful women role models in STEM to counter negative
+stereotypes.
+
+Girls Who Code, a national organization whose mission
+is to close the gender gap in technology
+[@saujani_girls_2015], was founded in 2012.
+Because of our personal experiences and the paucity of women in our field, we
+began a Girls Who Code student organization at the University of Michigan in 2017.
+For the past four academic years we have registered annually as a recognized
+Girls Who Code Club because the national organization provides name recognition,
+curriculum resources, guidance for a Capstone Impact Project, and a framework
+for launching a coding club.
+Participants in the Club attend weekly meetings at the University of Michigan (when
+the club is run in person rather than virtually), and are thus largely high school
+women from the Ann Arbor area.
+In 2019 we launched a Data Science Summer Experience that is unaffiliated with
+the national Girls Who Code organization. When held in person, the Summer Experience
```

Comment:
```suggestion
In 2019 we launched a Data Science Summer Experience program (not officially registered with
the national Girls Who Code organization). When held in person, the Summer Experience
```
Tweaking the wording since we still use GWC logos, maybe we don't even need the part in parentheses.

Reply:
Alternative idea below, what do you think?
@bnwolford @hayse1 
```suggestion
In 2019 we launched our own summer program, the Data Science Summer Experience. 
When held in person, the Summer Experience
```